provisioners/src/main/java/org/jboss/intersmash/provision/openshift/InfinispanOperatorProvisioner.java

```diff
@@ -144,6 +124,19 @@ public List<Pod> getPods() {
 				: Lists.emptyList();
 	}
 
+	public List<Pod> getInfinispanPods() {
+		return getInfinispanPods(getApplication().getName());
+	}
+
+	public static List<Pod> getInfinispanPods(final String clusterName) {
+		return OpenShiftProvisioner.openShift.inNamespace(OpenShiftConfig.namespace()).pods().list().getItems().stream().filter(
+				p -> p.getMetadata().getLabels().entrySet().stream()
```

Comment:
@fabiobrz are these labels assigned by the operator? is there any doc about this?

Reply:
You're right, I might need to back up this logic with resources. Thanks, fixing.

Reply:
@fabiobrz are you adding any comment here? I mean, should I expect one before closing the thread?

Reply:
Yep, I'll add one comment, thanks. The only reference about the `app=infinispan-pod` label I've found is in https://infinispan.org/docs/infinispan-operator/main/operator.html - although there it is mentioned just for the `affinity` aspects.

Reply:
Thank you @fabiobrz